cime_config/allactive/config_compsets.xml

```diff
@@ -353,6 +353,14 @@
   <lname>SSP585_EAM%CMIP6_ELM%CNPECACNTBC_MPASSI%SPUNUP_MPASO%SPUNUP_MOSART_SGLC_SWAV_BGC%BDRD</lname>
 </compset>
 
+<!-- SCREAM DYAMOND compsets -->
+<compset>
+  <alias>A_SCREAM-HR-DYAMOND2-2010</alias>
```

Comment:
This is a larger discussion point rather than a complaint about this compset: it is a bit oxymoronic to include 2010 *and* DYAMOND2 in the compset name since DYAMOND2 implies 2020, not 2010. Can we drop the 2010 from the end of this name? Note that our F compsets also have 2010 and DYAMOND2 in their name. @brhillman and @crterai ?

Reply:
@singhbalwinder - I think you can drop the "-2010" from the end of this compset name. Could you make that change and confirm it still runs? If so, I will integrate it today. It would be nice to have an initial condition/grid for this case, but I think that will take much more work (including get exec blessing to bug Luke and Jon) so should be a new PR.

Reply:
Certainly the 2010 can go away in the alias. And, really, we should probably take it out of the lname too, maybe replace "2010" with "2020" or "DYAMOND2". I think previously, we had the 2000, and later the 2010 in there because we were setting some parameters for the compset based on those things, like the use case files. Since we set specific use cases for the DYAMOND compset, I'm not sure this is needed. The other two places I can think of that would use this would be setting the run startdate, and CAM_CONFIG_OPTS. Our setting of CAM_CONFIG_OPTS for scream does not look for 2010 though, so that's not a problem, and RUN_STARTDATE is set by looking for the string `_EAM%SCREAM-[H,L]R-DYAMOND[1,2]`, so I think we are good there too. TL;DR, I think we can lose the 2010 altogether.